community-plugins.json

```diff
@@ -1,4 +1,22 @@
 [
+  {
+    "id": "rizzler",
+    "name": "rizzler",
+    "author": "@bihanmahadewa",
+    "description": "analyze your rizz, get feedback on how to be a better rizzler.",
+    "image": "/assets/plugin_images/rizzler.png",
+    "prompt": "",
+    "memories": false,
+    "chat": false,
+    "capabilities": [
+      "external_integration"
+    ],
+    "external_integration": {
+      "triggers_on": "memory_creation",
```

Comment:
Also, you might want to use "transcript_processed"event instead, so that it triggers during conversation, this would trigger only when the memory creates, after the conversation.